### PR TITLE
Add pyasn1-modules

### PR DESCRIPTION
**Pull request by jakirkham**
Generated with `conda skeleton pypi` and cleaned up to match our example.

### CODE REVIEW

**Comment by jakirkham** [comment]
Any thoughts on this package, @pmlandwehr? Any interest in helping maintain it?

Asking because of your interest in `pyasn1`.

**Comment by pmlandwehr** [comment]
If it doesn't build unless `pyasn1` is present at build time, then it should be fine otherwise.

**Comment by pmlandwehr** [comment]
Sure, I can help maintain.

File: recipes/pyasn1-modules/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyasn1 >=0.1.8
```
Comment (jakirkham) [inline recipes/pyasn1-modules/meta.yaml]: Yep, you're right. This should be removed.

Comment (jakirkham) [inline recipes/pyasn1-modules/meta.yaml]: Fixed.

**Comment by jakirkham** [comment]
Seems like this has done enough sitting. 😆 Self-merging so as to unblock some things.

**Pull request merged**